docs/NSW-Controllers.md

```diff
@@ -0,0 +1,30 @@
+## Installation
+
+First, you need to install [dkms-hid-nintendo](https://github.com/nicman23/dkms-hid-nintendo), a Nintendo HID kernel module:
+
+```shell
+git clone https://github.com/nicman23/dkms-hid-nintendo
+cd dkms-hid-nintendo
+sudo dkms add .
+sudo dkms build nintendo -v 3.0
+sudo dkms install nintendo -v 3.0
+```
+
+Then, you need [joycond](https://github.com/DanielOgorchock/joycond), a userspace driver which manages the controllers and exposes their motion inputs:
+
+
+```shell
+git clone https://github.com/DanielOgorchock/joycond.git
+cmake .
+sudo make install
+sudo systemctl enable --now joycond
+```
+## Usage
+
+Just like in a Ninendo Switch, after you connect it to the Raspberry Pi, you'll need to press triggers to select the position. After you press any button and the player indicator lights start blinking, do the following:
```

Comment:
_Ninteno_ should be _Nintendo_. 
Don't mention Raspberry Pi, since RetroPie works on other systems. 

It's not clear how you connect the controller to the system ? Do you have to pair it by Bluetooth first or, just like for PS3 controllers, you connect it via USB and pair it, then it will work wireless ?